app/src/main/java/ch/epfl/sdp/database/firebase/entity/EnemyForFirebase.java

```diff
@@ -1,13 +1,13 @@
 package ch.epfl.sdp.database.firebase.entity;
 
-import ch.epfl.sdp.geometry.GeoPoint;
+import ch.epfl.sdp.database.firebase.GeoPointForFirebase;
 
 /**
  * The in-game enemy entity to be stored in the cloud firebase
  */
 public class EnemyForFirebase {
     private int id;
-    private GeoPoint location;
+    private GeoPointForFirebase location;
```

Comment:
I think a good idea for the future is to have on class that adapts everything we need for Firebase (that's what other teams did that had 90% coverage). I'm not going to make you do it on this pull request but it's a good idea for a later pull request

Reply:
Yeah I agree with Amro as well